Lib/test/test_pydoc.py

```diff
@@ -449,6 +449,25 @@ def test_issue8225(self):
         result, doc_loc = get_pydoc_text(xml.etree)
         self.assertEqual(doc_loc, "", "MODULE DOCS incorrectly includes a link")
 
+    def test_issue41287(self):
```

Comment:
I just realized this is in `test_pydoc`, which should only test the `pydoc` module. Can you put it in `Lib/test/test_property.py` instead?